src/imdb_detector.py

```diff
@@ -98,6 +106,27 @@ def _is_valid_roman_numeral(self, cell: str) -> int:
             return 0
         return 1
 
+    def is_not_a_series_years(self, value: str) -> bool:
+        """
+        Check if a string is not a year (4-digit number).
+        """
+        tokens = self.tokenizer.tokenize_cell(value)
+        for token in tokens:
+            if not token.isdigit() or len(token) != 4:
+                return token
+        return 0
+
+    def is_not_a_production_year(self, value: str) -> bool:
+        """
+        Check if a string is not a production year (4-digit number).
```

Comment:
maybe add as a comment, that the 4-digit number has the format 2020.0, otherwise it is confusing that we need 2 different methods for 4-digit numbers